sample-new-architecture/app.json

```
@@ -0,0 +1,4 @@
+{
+  "name": "sampleNewArchitecture",
+  "displayName": "sampleNewArchitecture"
+}
```

```suggestion
}

```

Double check all the other files that missed the end new line